akka-actor-testkit-typed/src/main/scala/akka/actor/testkit/typed/scaladsl/ActorTestKit.scala

```diff
@@ -173,6 +175,16 @@ final class ActorTestKit private[akka] (val name: String, val config: Config, se
   def spawn[T](behavior: Behavior[T], name: String, props: Props): ActorRef[T] =
     Await.result(internalSystem ? (ActorTestKitGuardian.SpawnActor(name, behavior, _, props)), timeout.duration)
 
+  /**
+   * Stop the actor under test and wait until it terminates.
```

Comment:
It should also be documented that it can only be used for stopping of actors that were spawned by this `ActorTestKit`.